### PR TITLE
Implement isChecking

### PR DESCRIPTION
**Pull request by jennazee**
Addresses #22

### CODE REVIEW

**Comment by HipsterBrown** [comment]
:+1:

**Comment by gr2m** [comment]
Can you add a check in https://github.com/hoodiehq/hoodie-client-connection-status/blob/master/tests/integration/instance.js that `connectionStatus.isChecking` is a function?

Besides that is great

File: README.md [hunk 1]
```diff
@@ -337,7 +331,7 @@ connectionStatus.reset(options).then(function () {
 
 ---
 
-🐕 **TO BE DONE**: [abort pending request](https://github.com/hoodiehq/hoodie-client-connection-status/issues/21)
```
Comment (gr2m) [inline README.md]: can you leave the spaces, they are needed for line break

**Comment by jennazee** [comment]
@gr2m p.s. did those things!

**Pull request merged**